### PR TITLE
Follow-up PR to #122

### PR DESCRIPTION
**Pull request by 1010Tom**

### CODE REVIEW

**Review by holzeis — APPROVED** [review]
Thank you 👍 

No sure about the `new_app` and `new_coordinator` apis, but we can change that later if needed.

File: crates/ln-dlc-node/src/node.rs [hunk 1]
```diff
@@ -112,17 +112,70 @@ impl Display for NodeInfo {
...
+            network,
+            data_dir,
+            address,
+            electrs_origin,
+            seed,
+            ephemeral_randomness,
+            user_config,
+        )
+        .await
+    }
```
Comment (luckysori) [inline crates/ln-dlc-node/src/node.rs:164]: I suppose we'll eventually move this code to the specific crates, but this is a good first step. Thank you!

File: crates/ln-dlc-node/src/tests/system_tests/mod.rs [hunk 2]
```diff
@@ -139,19 +139,29 @@ pub(crate) async fn setup_ln_node(test_dir: &Path, node_name: &str) -> Node {
...
-        node_name.to_string(),
-        Network::Regtest,
-        data_dir.as_path(),
-        address,
-        ELECTRS_ORIGIN.to_string(),
-        seed,
-        ephemeral_randomness,
-    )
-    .await
+    if is_coordinator {
```
Comment (luckysori) [inline crates/ln-dlc-node/src/tests/system_tests/mod.rs:142]: Hahahaha nooooo. Two functions better than a boolean flag? :stuck_out_tongue:

**Review by luckysori — APPROVED** [review]
Thanks for addressing the suggestions!

File: crates/ln-dlc-node/src/tests/system_tests/mod.rs (hunk 2 above)
Comment (bonomat) [reply crates/ln-dlc-node/src/tests/system_tests/mod.rs:142]: With two functions we would have to duplicate so much stuff, so I went for the easier one :D

**Pull request merged**